stdlib/camlinternalFormat.ml

```diff
@@ -286,11 +291,12 @@ let char_of_iconv iconv = match iconv with
   | Int_Co -> 'o' | Int_u | Int_Cu -> 'u'
 
 (* Convert a float conversion to char. *)
-let char_of_fconv fconv = match fconv with
-  | Float_f | Float_pf | Float_sf -> 'f' | Float_e | Float_pe | Float_se -> 'e'
-  | Float_E | Float_pE | Float_sE -> 'E' | Float_g | Float_pg | Float_sg -> 'g'
-  | Float_G | Float_pG | Float_sG -> 'G' | Float_F -> 'F'
-  | Float_h | Float_ph | Float_sh -> 'h' | Float_H | Float_pH | Float_sH -> 'H'
+(* `cF' will be 'F' for displaying format and 'g' to call libc printf *)
+let char_of_fconv cF fconv = match snd fconv with
```

Comment:
I think that it would be clearer, at call sites, if `cF` was a labelled argument. (I can't think of a much better name than `cF` or `conv_F`, but maybe you can.)

Reply:
I agree, done. No better idea for the argument name.